utils/ChromiumDownloader.js

```diff
@@ -216,3 +221,16 @@ function downloadFile(url, destinationPath, progressCallback) {
 function extractZip(zipPath, folderPath) {
   return new Promise(fulfill => extract(zipPath, {dir: folderPath}, fulfill));
 }
+
+function applyAgent(opt)
+{
+  if (process.env.http_proxy || process.env.https_proxy)
+  {
+    let parsedProxy = process.env.http_proxy ? URL.parse(process.env.http_proxy) : URL.parse(process.env.https_proxy);
```

Comment:
A few questions:
1. we should support capitalized env variables as well (and camel-cased, probably). There's also the NO_PROXY env variable, not sure how often it is used. Maybe rely on https://github.com/Rob--W/proxy-from-env to parse environment? 
2. we only do a single request to https url - do we care about `http_proxy` at all?